### PR TITLE
eTarget Bid Adapter: add "getMetaData" function to adapter

### PR DESCRIPTION
**Pull request by etargetse**
getMetaData used to gether useful information from publisher site

## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@etargetse looks like that worked :) Don't forget to add back in the test too if you could

File: modules/etargetBidAdapter.js
Comment (patmmccann) [inline modules/etargetBidAdapter.js:64]: Moving the request here,  please also gather this info from the ortb2 object

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by patmmccann** [comment]
Please also fix #6650 ; it's being enforced imminently

File: modules/etargetBidAdapter.js
Comment (etargetse) [reply modules/etargetBidAdapter.js:64]: problem is that ortb2 is available only when portal will fill info into this object, what we want to achieve is to gather info every time our adapter will be used. Is there any leagal/technical problem with our implementation? Why we cant collect info by ourselfs?

**Comment by etargetse** [comment]
> Please also fix #6650 ; it's being enforced imminently

working on it

File: modules/etargetBidAdapter.js
Comment (etargetse) [reply modules/etargetBidAdapter.js:64]: > Please also fix #6650 ; it's being enforced imminently

I started working on this feature but i need to create new fork because when i edit things on old fork then circleci have always problems with somethink. But i cant create new fork before this pull requerst is closed. So can you please merge this new feature if there is no real reason why not to merge? THENKS.

**Comment by etargetse** [comment]
Hey guys, we are in bit hurry, please if our code has no legal or coding errors can you please review our adapter and merge it into master repo? Thanks.

**Comment by ChrisHuie** [comment]
@etargetse you still need to add back the unit tests before this pr can get approved please

**Comment by etargetse** [comment]
i am closing this pull request because when i update test script then it will fail because this new feature is not included

**Pull request closed without merging**
